如果命令输入错误，报错信息不能正确反馈到前端

后端会报错 AttributeError: 'AccessControlXXXXError' object has no attribute 'message'
然后前端QQ会显示`该消息类型暂不支持查看`
触发方式很简单，只需要输入错误的指令即可，比如`--`少打了一个，只打了一个`-`

bot回复的消息：（比如--sbj少打了一个，只打了一个-）
![image](https://github.com/user-attachments/assets/53c8c16c-8e3e-48d4-b398-fc444438877e)
nonebot那边的error：
``` bash
01-02 09:14:56 [ERROR] nonebot_plugin_access_control | 'AccessControlBadRequestError' object has no attribute 'message'
Traceback (most recent call last):
  File "/usr/local/lib/python3.12/site-packages/nonebot_plugin_access_control/handler/utils/errors.py", line 46, in run_excepting
    yield
  File "/usr/local/lib/python3.12/site-packages/nonebot_plugin_access_control/handler/__init__.py", line 37, in _handing_error
    yield
  File "/usr/local/lib/python3.12/site-packages/nonebot_plugin_access_control/handler/__init__.py", line 60, in handle_ac
    raise AccessControlBadRequestError("命令格式错误")
nonebot_plugin_access_control_api.errors.AccessControlBadRequestError: 命令格式错误

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/app/bot.py", line 15, in <module>
    nonebot.run()
  File "/usr/local/lib/python3.12/site-packages/nonebot/__init__.py", line 337, in run
    get_driver().run(*args, **kwargs)
  File "/usr/local/lib/python3.12/site-packages/nonebot/drivers/none.py", line 61, in run
    anyio.run(self._serve)
  File "/usr/local/lib/python3.12/site-packages/anyio/_core/_eventloop.py", line 74, in run
    return async_backend.run(func, args, {}, backend_options)
  File "/usr/local/lib/python3.12/site-packages/anyio/_backends/_asyncio.py", line 2347, in run
    return runner.run(wrapper())
  File "/usr/local/lib/python3.12/asyncio/runners.py", line 118, in run
    return self._loop.run_until_complete(task)
  File "/usr/local/lib/python3.12/asyncio/base_events.py", line 673, in run_until_complete
    self.run_forever()
  File "/usr/local/lib/python3.12/asyncio/base_events.py", line 640, in run_forever
    self._run_once()
  File "/usr/local/lib/python3.12/asyncio/base_events.py", line 1992, in _run_once
    handle._run()
  File "/usr/local/lib/python3.12/asyncio/events.py", line 88, in _run
    self._context.run(self._callback, *self._args)
  File "/usr/local/lib/python3.12/site-packages/nonebot/utils.py", line 254, in run_coro_with_shield
    return await coro
  File "/usr/local/lib/python3.12/site-packages/nonebot/message.py", line 506, in check_and_run_matcher
    await _run_matcher(
  File "/usr/local/lib/python3.12/site-packages/nonebot/message.py", line 458, in _run_matcher
    await matcher.run(bot, event, state, stack, dependency_cache)
  File "/usr/local/lib/python3.12/site-packages/nonebot/internal/matcher/matcher.py", line 926, in run
    await self.simple_run(bot, event, state, stack, dependency_cache)
  File "/usr/local/lib/python3.12/site-packages/nonebot/internal/matcher/matcher.py", line 863, in simple_run
    await handler(
  File "/usr/local/lib/python3.12/site-packages/nonebot/dependencies/__init__.py", line 113, in __call__
    return await cast(Callable[..., Awaitable[R]], self.call)(**values)
  File "/usr/local/lib/python3.12/site-packages/nonebot_plugin_access_control/matcher.py", line 40, in handle_cmd
    await handle_ac(f, "ac " + cmd_body)
  File "/usr/local/lib/python3.12/site-packages/nonebot_plugin_access_control/handler/__init__.py", line 43, in handle_ac
    async with _handing_error(fout):
  File "/usr/local/lib/python3.12/contextlib.py", line 231, in __aexit__
    await self.gen.athrow(value)
> File "/usr/local/lib/python3.12/site-packages/nonebot_plugin_access_control/handler/__init__.py", line 36, in _handing_error
    async with error_handlers.run_excepting(fout.write, reraise_unhandled=True):
  File "/usr/local/lib/python3.12/contextlib.py", line 231, in __aexit__
    await self.gen.athrow(value)
  File "/usr/local/lib/python3.12/site-packages/nonebot_plugin_access_control/handler/utils/errors.py", line 70, in run_excepting
    msg = e.message
AttributeError: 'AccessControlBadRequestError' object has no attribute 'message'

```

--- a/.github/workflows/pr-check.yml
+++ b/.github/workflows/pr-check.yml
@@ -25,7 +25,7 @@ jobs:
         python-version: ${{ matrix.python-version }}
     - name: Install requirements
       run: |
-        pip install poetry
+        pip install poetry==1.8.5
         poetry export --with=dev --without-hashes -o requirements.txt
         pip install -r requirements.txt
         pip install "${{ matrix.pydantic }}"

--- a/pyproject.toml
+++ b/pyproject.toml
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "nonebot-plugin-access-control"
-version = "1.2.2"
+version = "1.2.3"
 description = ""
 authors = ["ssttkkl <huang.wen.long@hotmail.com>"]
 license = "MIT"

--- a/src/nonebot_plugin_access_control/handler/utils/errors.py
+++ b/src/nonebot_plugin_access_control/handler/utils/errors.py
@@ -67,7 +67,7 @@ class ErrorHandlers:
                         return
 
             if isinstance(e, (AccessControlBadRequestError, AccessControlQueryError)):
-                msg = e.message
+                msg = str(e)
 
                 matcher = None
                 try:

